index.html

```diff
@@ -98,231 +93,34 @@ <h1 class="suptext">LET'S CONNECT</h1>
-        <div class="p-layout">
-          <h1 class="portfolio-head2">Multi-Post<br />Stories</h1>
-          <h1 class="portfolio-head2d">Uber Navigation</h1>
-          <ul class="frame">
-            <li>
-              <h2 class="canopy">CANOPY</h2>
-            </li>
-            <li>
-              <img src="Counter.png" alt="1" />
-            </li>
-            <li>
-              <h2 class="backend">Back End Dev</h2>
-            </li>
-            <li>
-              <img src="Counter.png" alt="2" />
-            </li>
-            <li>
-              <h2 class="year">2015</h2>
-            </li>
-          </ul>
-          <ul class="framed">
-            <li>
-              <h2 class="canopy">Uber</h2>
-            </li>
-            <li>
-              <img src="Counter.png" alt="3" />
-            </li>
-            <li>
-              <h2 class="backend">Lead Developer</h2>
-            </li>
-            <li>
-              <img src="Counter.png" alt="4" />
-            </li>
-            <li>
-              <h2 class="year">2015</h2>
-            </li>
-          </ul>
-
-          <p class="p-text">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <p class="p-textd">
-            A smart assistant to make driving more safe, efficient, and fun by
-            unlocking your most expensive computer: your car.
-          </p>
-          <ul class="lang">
-            <li class="p-html"><a class="a-lang" href="">html</a></li>
-            <li class="p-css"><a class="a-lang" href="">css</a></li>
-            <li class="p-javas"><a class="a-lang" href="">javaScript</a></li>
-          </ul>
-          <button class="p-button" type="submit">See Project</button>
+      <div class="bg-modal">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h1 class="modal-head" id="modal-head">Tonic</h1>
+            <div class="close">+</div>
+          </div>
+          <ul class="frame" id="expd"></ul>
+          <div class="modal-imagec">
+            <img src="SnapshotPortfoliodesk.png" class="modal-image" alt="" />
+          </div>
+          <div class="modal-layout">
+            <p class="p-modal" id="p-modald">Test</p>
+            <div class="r-modal">
+              <ul class="lang" id="lan"></ul>
+              <div class="b-layout">
+                <a class="live p-button"
+                  >See live <span><i class="fa fa-external-link"></i></span
+                ></a>
+                <a class="source p-button"
+                  >See source <span><i class="fa fa-github"></i></span
+                ></a>
+              </div>
+            </div>
+          </div>
         </div>
-      </section>
+      </div>
     </div>
+    <ul class="card-list" id="cards"></ul>
     <div id="about" class="grid-container2">
       <section class="card3">
         <h1 class="a-head">About<br />Myself</h1>
```

Comment:
Please add linter config for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link.